paket.dependencies

```diff
@@ -1,4 +1,6 @@
+framework: auto-detect
+
 source https://api.nuget.org/v3/index.json
 
 nuget Elmish 2.0.0-beta-4 prerelease
-nuget FSharp.Core 4.5.2
+nuget FSharp.Core >= 4.5.2
```

Comment:
I'd prefer to depend on the lowest possible FSharp.Core. Also see elmish/elmish#146.

Reply:
In any case, Elmish core requires >= 4.5.2 so it's kinda a moot point. But I lowered it in case Elmish lowers it. No need to require more than we need, even if our dependencies do.